tests/acceptance/features/bootstrap/EmailContext.php

```diff
@@ -61,9 +61,12 @@ public function assertThatEmailContains($address, PyStringNode $content) {
 		$expectedContent = $this->featureContext->substituteInLineCodes(
 			$expectedContent
 		);
+		$emailBody = EmailHelper::getBodyOfLastEmail($this->localMailhogUrl, $address);
 		Assert::assertContains(
 			$expectedContent,
-			EmailHelper::getBodyOfLastEmail($this->localMailhogUrl, $address)
+			$emailBody,
+			"The email address {$address} should have received an email with the body containing {$content}
```

Comment:
```suggestion
			"The email address {$address} should have received an email with the body containing {$expectedContent}
```

`$content` is a `PyStringNode` - does that have a `toString` method that will substitute OK in a text message?

Try a scenario locally whit a wrong expectation and see what happens.